### PR TITLE
Delete property command

### LINKED ISSUE #84: Complete Delete Property Command

### PR DESCRIPTION
**Pull request by jolwnn**

### CODE REVIEW

**Review by thony-ui — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/DeletePropertyCommand.java [hunk 1]
```diff
@@ -46,7 +48,16 @@ public DeletePropertyCommand(PostalCode postalCode, Unit unitNumber) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        return new CommandResult(MESSAGE_DELETE_PROPERTY_SUCCESS);
+
+        Property propertyToDelete = model.getFilteredPropertyList().stream()
```
Comment (yongkheehou) [inline src/main/java/seedu/address/logic/commands/DeletePropertyCommand.java:52]: caveat that this looks like a violation of the Law of Demeter. Maybe we should add a method `toDelete(postalCode, unitNumber)` instead. I think DeleteBuyerCommand was implemented similarly though. We can leave it here first for v1.3 I think

File: src/test/java/seedu/address/logic/commands/DeletePropertyCommandTest.java [hunk 2]
```diff
@@ -20,6 +32,57 @@
...
 
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), getTypicalPropertyBook(),
+            getTypicalClientBook());
+
+    @Test
+    public void execute_validPostalCodeAndUnitNumber_success() {
+        PostalCode postalCode = new PostalCode(VALID_POSTALCODE_ADMIRALTY);
+        Unit unit = new Unit(VALID_UNIT_ADMIRALTY);
+        Property propertyToDelete = model.getFilteredPropertyList().stream()
+                .filter(property -> property.getPostalCode().equals(postalCode)
```
Comment (yongkheehou) [inline src/test/java/seedu/address/logic/commands/DeletePropertyCommandTest.java:43]: same comment about LoD here

File: src/test/java/seedu/address/testutil/TypicalProperty.java [hunk 3]
```diff
@@ -19,8 +19,8 @@ public class TypicalProperty {
      */
     public static PropertyBook getTypicalPropertyBook() {
         PropertyBook ab = new PropertyBook();
-        for (Property person : getTypicalProperties()) {
-            ab.addProperty(person);
+        for (Property property : getTypicalProperties()) {
```
Comment (yongkheehou) [inline src/test/java/seedu/address/testutil/TypicalProperty.java:22]: nice catch LOL

**Review by yongkheehou — APPROVED** [review]
LGTM

**Pull request merged**